Sources/SwiftKueryMySQL/MySQLConnection.swift

```diff
@@ -45,6 +45,9 @@ public class MySQLConnection: Connection {
     private let clientFlag: UInt
     private let characterSet: String
     private let reconnect: Bool
+    
+    /// Connection timeout in seconds
```

Comment:
Why not keep this as milliseconds and do the conversion in `connect()`?

Reply:
OK. I Will fix it.